external/src/main/java/org/hawkular/alerts/AlertsStandalone.java

```diff
@@ -153,36 +153,34 @@ public AlertsStandalone() {
         status.setPartitionManager(partitionManager);
     }
 
-    public CompletableFuture<Void> init() {
-        return CompletableFuture.runAsync(() -> {
-            if (this.ispnReindex) {
-                log.info("Reindexing of Infinispan [backend] started.");
-                status.setReindexing(true);
-                long startReindex = System.currentTimeMillis();
-                SearchManager searchManager = Search
-                        .getSearchManager(IspnCacheManager.getCacheManager().getCache("backend"));
-                DistributedExecutorMassIndexer massIndexer = (DistributedExecutorMassIndexer) searchManager.getMassIndexer();
-                // Lets block instead of async
-                massIndexer.start();
-                long stopReindex = System.currentTimeMillis();
-                log.info("Reindexing of Infinispan [backend] completed in [" + (stopReindex - startReindex) + " ms]");
-                ispnReindex = false;
-                status.setReindexing(false);
-            }
-            // Initialization needs order and needs to be done after reindexing
-            ispnAlerts.init();
-            ispnDefinitions.init();
-            ispnActions.init();
-            adminService.init();
-
-            partitionManager.init();
-            alertsContext.init();
-            dataDrivenGroupCacheManager.init();
-            actionsCacheManager.init();
-            publishCacheManager.init();
-            extensions.init();
-            engine.initServices();
-        });
+    public void init() {
```

Comment:
This method is only called by `AlertStarter#initialize` which doesn't seem async execution. It's ok to drop the `CompletableFuture<Void>` return type.